README.md

```diff
@@ -105,6 +105,7 @@ Slow Reader is a local-first app. Clients do most of the work, and the server ju
 - [`api/`](./api/): types and constants shared between clients and server.
 - [`docs/`](./docs/): guides for developers.
 - [`scripts/`](./scripts/): scripts to test project and configure Google Cloud. Check the script’s descriptions for further details.
+- [`loader-tests/`](./loader-tests/): loader testing tool for OPML files.
```

Comment:
```suggestion
- [`loader-tests/`](./loader-tests/): integration tests for each social network or news format.
```

`loader` is our internal term. A new developer will not understand what it is.

Also it is not only for OPML files, since later we will do much more integration tests in that dir.